hoot-core/src/main/cpp/hoot/core/conflate/polygon/BuildingMerger.cpp

```diff
@@ -243,6 +245,8 @@ void BuildingMerger::apply(const OsmMapPtr& map, vector<pair<ElementId, ElementI
         scrap->getElementId() << "...");
     }
 
+    // TODO: need to explain how this tag merging differs from that done in buildBuilding and
```

Comment:
`BuildingMerger` tag merging is a bit hard to follow. Once I completely understand it, I'll try to update the comments here.